arangod/Cluster/RestClusterHandler.cpp

```diff
@@ -42,32 +42,41 @@ RestClusterHandler::RestClusterHandler(GeneralRequest* request, GeneralResponse*
     : RestBaseHandler(request, response) {}
 
 RestStatus RestClusterHandler::execute() {
-  if (_request->requestType() != RequestType::GET) {
-    generateError(rest::ResponseCode::METHOD_NOT_ALLOWED, TRI_ERROR_HTTP_METHOD_NOT_ALLOWED,
-                  "only the GET method is allowed");
-    return RestStatus::DONE;
-  }
+  ServerState::RoleEnum role = ServerState::instance()->getRole();
+
+  if (ServerState::instance()->isCoordinator(role) ||
+      (ServerState::instance()->isSingleServer(role) && AgencyCommManager::isEnabled())) {
```

Comment:
doesn't handleCommandEndpoints check the same conditions ?
Also this handler is only registered if the cluster feature is enabled

Reply:
However, the RestHandler is still callable in active failover mode. That I verified.

Reply:
but it should not be callable in single server mode